modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -132,29 +132,41 @@ image::sso-configuration-tutorials/okta-sign-on-tab.svg[title="Okta's sign-on ta
 
 This example shows how to configure Entra ID for authentication and authorization using an access token.
 
-. Set parameters to be `access_token`:
+. Assuming that you have already created and registered your SSO application in Azure, you should now configure a token in Entra ID using the *Token configuration* tab.
+
+. Next step is to go to the **Expose an API** tab and click the **Add a Scope** button. 
++
+You need to know your Essentials located in the GUID -- Globally Unique Identifier. 
++
+.The GUID is used to identify specific resources or instances within Azure. You can find it on the app registration page.
+image::sso-configuration-tutorials/azure-id.svg[]
++
+If you click the **Add a Scope** button for the first time, you see a new window stating that you need to add an _Application ID URI_ before proceeding:
++
+[source]
+----
+api://4376dc8b-b5af-424f-9ada-c1c1b2d416b9/access-token
```

Comment:
I think the screenshot is enough. We don't need to have it in a code format as well as it is different for every app.